third_party/crashpad/crashpad/util/net/http_transport_socket.cc

```diff
@@ -329,7 +329,8 @@ base::ScopedFD CreateSocket(const std::string& hostname,
         } else if (WaitUntilSocketIsReady(result.get())) {
           return result;
         }
-        return base::ScopedFD();
+        continue;
```

Comment:
Instead of commenting out the upstream code (from Chromium), please make the customization using preprocessor directives:

```
#if defined(STARBOARD) || defined(NATIVE_TARGET_BUILD)
        continue;
#else
        return base::ScopedFD();
#endif
```

Please see other examples of this in this file.

Reply:
@hlwarriner :Many thks for your comment.
We will use the the macro to control it.

Reply:
@johnxwork : 
Sorry for mistake.
This is my first time submit the code to cobalt.
About the issue link ,https://partnerissuetracker.corp.google.com/u/0/issues/323792488.
Because did not know your mail address,i can not add you into the cc list.
Could you let danleemh(also from google) add your into the cc list.

In paralley, let me decrible the issue detailly.
We found sometime get the ap list from the addrinfo. first member is IPV6 and second member is IPV4.
Due to our platform network environment did not support IPV6, it caused the createSocket fail.and then the test case fail.
Therefore we want to traverse the ap list,try to find the ipv4.
Many thks~.

Reply:
Hi @Hyobo , JohnX and Holden are now in cc list of b/323792488

Reply:
@jonastsai :
Many thks.

Reply:
No problem! And thanks for making the change to use the macros.

I have a follow-up question.

> Due to our platform network environment did not support IPV6, it caused the createSocket fail

Where does the `createSocket` call occur?